pallets/parachain-staking/src/lib.rs

```diff
@@ -2098,11 +2987,8 @@ pub mod pallet {
 				Error::<T>::AlreadyActive
 			);
 			<CandidatePool<T>>::put(candidates);
-			<CandidateState<T>>::insert(&collator, state);
-			Self::deposit_event(Event::CandidateBackOnline(
-				<Round<T>>::get().current,
-				collator,
-			));
+			<CandidateInfo<T>>::insert(&collator, state);
+			Self::deposit_event(Event::CandidateBackOnline(collator));
```

Comment:
This is a small optimization that was unrelated to this PR but included anyway. There is no need to emit round number in event.